app/src/main/res/values-land/dimens.xml

```diff
@@ -384,10 +385,14 @@
   <dimen name="profile_rename_activity_profile_rename_input_margin_end">168dp</dimen>
   <dimen name="profile_rename_activity_profile_rename_save_button_margin_end">168dp</dimen>
 
+  <!-- Story Fragment- land dimens -->
+  <dimen name="story_fragment_activity_padding_bottom">60dp</dimen>
+  
   <!-- SectionTitleFragment -->
   <dimen name="section_title_divider_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_start">72dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">72dp</dimen>
 
+
```

Comment:
Remove this extra blank line.

Reply:
@shivambh12 This is pending. Also please make sure you reply to all the comments.

Reply:
okk,I will remove it